Documentation/install/upgrade.rst

```diff
@@ -313,6 +313,26 @@ Removed options
 
 * All code associated with ``monitor v1.0`` socket handling has been removed.
 
+.. _1.7_upgrade_notes:
+
+1.7 Upgrade Notes
```

Comment:
The section immediately above this already has the 1.7 Upgrade notes title.

Reply:
Fixed